pkg/report/template.go

```diff
@@ -15,6 +16,10 @@ import (
 const (
 	assetsDir    = "assets"
 	templatesDir = "templates"
+
+	statusSkipped       = 0
+	testSkippedWording  = "Dgoss tests were not run because the docker image fail to built correctly"
```

Comment:
Dgoss is the name of a bash script from the Goss project. We don't even use the Dgoss script anywhere

```suggestion
	testSkippedWording  = "Goss tests skipped because the docker image failed to build"
```